sympy/integrals/integrals.py

```diff
@@ -153,6 +153,60 @@ def _eval_is_zero(self):
         if self.function.is_zero is False and got_none is False:
             return False
 
+    def is_divergent(self):
+        """
+        Check if the intergal is divergent for the integration limits.
+
+        Examples
+        ========
+
+        >>> from sympy import Integral, oo
+        >>> from sympy.abc import x
+        >>> f = x**2
+        >>> Integral(f,(x, 0, oo)).is_divergent()
+        True
+        >>> Integral(f,(x, )).is_divergent()
+        True
+        >>> f = x**3
+        >>> Integral(f,(x,oo)).is_divergent()
+        True
+        >>> f = x**3
+        >>> print(Integral(f,(x, )).is_divergent())
+        None
```

Comment:
Convergence and divergence is not defined for indefinite integration as far as I'm aware. 

Also, if the second argument is a 2-tuple, then that represents indefinite integration with the symbol of integration being replaced by the second value in the tuple. This means `Integral(f,(x, oo))` does not represent an integral from 0 to oo. If `f = exp(-x)` then the result of `Integral(f,(x, oo))` is 0 while `Integral(f,(x, 0, oo))` is 1.